### DISCUSSION TITLE
Cypress test keep looping when I run test on localhost

### DISCUSSION THREAD

**Discussion by madebymt** [cypress-io/cypress#29251] [Component Testing]
Version:
- Cypress: Version 13.2.0 (13.2.0.2353610)
- Chrome: v123
- Mac: MacOS Ventura v13.5.1

How to reproduce:
1.  I open cypress app on my mac
2. Select the repo I want to run cypress test
3. Take me to the `http://localhost:58155/__/#/specs` page to select the test I want to run
4. After I click it, I get this infinity loop for my test.
![2024-04-02 15 27 28](https://github.com/cypress-io/cypress/assets/26178313/ebd5998b-815a-44e9-9de3-d9a7c1ba3426)

**My code:**
home.cy.js
```
describe('Default load page without token', () => {
  it('displays error page with try again', () => {
    cy.visit('http://localhost:8080');
    cy.wait(2000);
    cy.get('div[type="Button"]').should('have.length', 3);
  });
});

```
cypress config
```
module.exports = defineConfig({
  retries: 2,
  video: true,
  defaultCommandTimeout: 5000,
  viewportWidth: 1920,
  viewportHeight: 1080,

  // Viewport settings overridden for component tests
  component: {
    viewportWidth: 1920,
    viewportHeight: 1080,
  },
});
```
I tried to comment out all my `cypress.config` file, no changes, I only have one test.
Thanks for the help.

Ruby

**Comment by alasdairrobinson** [comment]
Are any files being written to during the test run? I've seen before in a test that modified a file in the cypress watched files list. I stopped it by setting watchForFileChanges to false in the config file.

**Comment by mendescamara** [comment]
I solved this problem by adding the config `chromeWebSecurity: true` in `cypress.config.ts`

The problem occurs because Cypress is on **https**.
If you manually change the url to **http** the problem does not happen.
